packages/dds/tree/src/test/time/tableTree.spec.ts

```diff
@@ -0,0 +1,754 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
```

Comment:
It would probably be a good idea to add a comment at the top of this file noting that we have counterpart tests for SharedMatrix in the `matrix` package, and that changes made here should likely be made there as well.

And similar comments in the matrix tests :)